src/Bicep.Core/Bicep.Core.csproj

```diff
@@ -25,10 +25,10 @@
     <PackageReference Include="JsonPatch.Net" Version="2.0.4" />
     <PackageReference Include="JsonPath.Net" Version="0.2.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="6.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="7.0.2" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
```

Comment:
I think this update may be causing the same issue that's blocking https://github.com/Azure/bicep/pull/8963. I believe that should be resolved by https://github.com/Azure/bicep-types/pull/56, which looks like it has been merged but not yet released.

Reply:
Thanks for pointing that out btw. I just tested it locally and it fixes the issue. I will get the types package released.

Reply:
Fixed